test/TestAssets/CoverletCoverageTestProject/CoverletCoverageTestProject.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Imports Common TestAssets props. -->
+  <Import Project="..\..\..\scripts\build\TestAssets.props" />
+  <Import Project="..\..\..\scripts\build\TestPlatform.Dependencies.props" />
+  <!-- Package dependency versions -->
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="$(NETTestSdkVersion)" />
+    <PackageReference Include="xunit" Version="2.4.0" />
```

Comment:
Similarly to the test.sdk version there should be variables for the other dependencies, please use them.

Reply:
ok will do!

Reply:
done

Reply:
aaaaand the coverlet version?

Reply:
not sure if it makes sense...this smoke test check vstest side...I mean we only test if we're replacing base path correctly.
BTW if you want we can fix also that version, but coverlet side integration is tested on our side https://github.com/tonerdo/coverlet/blob/master/test/coverlet.integration.tests/Collectors.cs
Let me know.

Reply:
The purpose of those variables is that we can easily update the versions accross all the test assets to the same version (unless they use some old specific version to test that we don't break compatibility with older release). If you don't use that variable then we might forget to update it, or it will be difficult to go back and forth among versions. So please use it :)

Reply:
Ok will do

Reply:
added `CoverletCoverageVersion` var